src/components/ToDo.js

```diff
@@ -43,20 +49,37 @@ const ToDo = () => {
       },
     ]);
     setInputText("");
+    handleAlert({ type: "success", text: "successfully added" });
+  };
+
+  const handleAlert = ({ type, text }) => {
+    setAlert({ show: true, type, text });
+    setTimeout(() => {
+      setAlert({ show: false });
+    }, 7000);
   };
 
   const clearItems = () => {
     setTodos([]);
+    handleAlert({ type: "danger", text: "all items deleted" });
   };
 
-  const deleteItem = id => {
-    let temp = todos.filter(item => item.id !== id);
+  const deleteItem = (id) => {
+    let temp = todos.filter((item) => item.id !== id);
     setTodos(temp);
+    handleAlert({ type: "danger", text: "item deleted" });
+  };
+
+  const editItem = (id) => {
+    //console.log(id);
+    const singleItem = todos.find((item) => item.id === id);
+    console.log(singleItem);
```

Comment:
Function name should be`handleEditItem` and it does nothing at all. Its incomplete.

Reply:
Malai edit garnai aayena, I am checking a tutorial and will fix this